docs/src/main/asciidoc/_admin-guide/subsystem-configuration/EE_Application_Deployment_Configuration.adoc

```diff
@@ -39,6 +39,75 @@ The optional `meta-inf` attribute, which defaults to `true`, indicates
 if the Module's `META-INF` path should be available to the deployment's
 class loader.
 
+[[global-directory]]
+== Global Directory
+Global modules can be used to shared common libraries across all deployed applications, but it could be unpractical if the name of a shared library changes very often or if there are many libraries you want to share. Both cases will require changes in the underlying module.xml that represents this global module.
+
+The EE subsystem allows the configuration of a global directory, which represents a directory tree scanned automatically to include .jar files and resources as a single additional dependency. This module dependency is added as a system dependency on each deployed application. Basically, with a global directory, you will be relying on WildFly to automatize the maintenance and configuration of a JBoss module that represents the jar files and resources of a specific directory.
+
+You can configure a global directory using the following operation:
+
+[source,xml,options="nowrap"]
+----
+  [standalone@localhost:9990 /] /subsystem=ee/global-directory=my-common-libs:add(path=lib, relative-to=jboss.home.dir)
+----
+
+The following attributes are available on the global-directory resource:
+
+* path: The path of the directory to scan. (Mandatory)
+* relative-to: The name of another previously named path, or of one of the standard paths provided by the system. (Optional)
+
+When a `global-directory` is created, the server establishes a JBoss module with one https://jboss-modules.github.io/jboss-modules/manual/#path-resource-loader[Path Resource Loader] created using 'path' and 'relative-to' attributes and one https://jboss-modules.github.io/jboss-modules/manual/#jar-resource-loader[Jar Resource loader] for each jar file included in this directory and subdirectories.
```

Comment:
s/JBoss module/JBoss Modules module/g

s/and subdirectories/and its subdirectories/g